primes.rb

```diff
@@ -1,6 +1,12 @@
+require 'prime'
+
 class Primes
-  def self.sum_to(limit = 100)
+  def self.sum_to(limit)
     # TODO - add your prime number solution here...
-    "I'm working on it!"
+    primeArray = []
+	Prime.each(limit) do |prime|
+ 		primeArray.push(prime)
+	end
+	@sum = primeArray.reduce(:+)
   end
 end
```

Comment:
This file (and several others in this PR) are missing terminating newline characters. Depending on your editor of choice you can usually auto-configure one to be added upon saving a file. Take a look here to understand why this is important: http://stackoverflow.com/a/729795